src/IoC/ServiceExtensions.cs

```diff
@@ -14,19 +17,28 @@
 
     public static class ServiceExtensions
     {
+        public static IServiceCollection AddServices(this IServiceCollection services)
+        {
+            return services
+                .AddTransient<IReportingHelper, ReportingHelper>()
+                .AddScoped<IStoragePlaceAuditReportService, StoragePlaceAuditReportService>();
+        }
+
         public static IServiceCollection AddFacadeServices(this IServiceCollection services)
         {
             return services
                 .AddSingleton<IRazorEngine, RazorEngine>()
                 .AddSingleton<ITemplateEngine, RazorTemplateEngine>()
+                .AddScoped<IStoragePlaceAuditReportFacadeService, StoragePlaceAuditReportFacadeService>()
                 .AddScoped<IAsyncFacadeService<Carrier, string, CarrierResource, CarrierUpdateResource, CarrierResource>, CarrierService>()
                 .AddScoped<IAsyncFacadeService<Country, string, CountryResource, CountryResource, CountryResource>, CountryService>();
         }
 
         public static IServiceCollection AddBuilders(this IServiceCollection services)
         {
             return services.AddScoped<IBuilder<Carrier>, CarrierResourceBuilder>()
-                .AddScoped<IBuilder<Country>, CountryResourceBuilder>();
+                .AddScoped<IBuilder<Country>, CountryResourceBuilder>()
+                .AddTransient<IReportReturnResourceBuilder, ReportReturnResourceBuilder>();
```

Comment:
scoped again? Unless there's some subtlety I'm missing

Reply:
will do. just habits

Reply:
I don't know how much different it makes really for our use case, just scoped seems more conventional for services like this. I think we used scoped back in the autofac days and then we (probably me) started using transient for no good reason when we ditched autofac....